db/db_impl.cc

```diff
@@ -1845,6 +1845,14 @@ bool DBImpl::GetProperty(ColumnFamilyHandle* column_family,
     InstrumentedMutexLock l(&mutex_);
     return cfd->internal_stats()->GetStringProperty(*property_info, property,
                                                     value);
+  } else if (property_info->is_immutable_db_statistic) {
+    assert(value != nullptr);
+    auto dbstats = immutable_db_options_.statistics.get();
```

Comment:
Google C++ Style discourage usage of `auto` like this: https://google.github.io/styleguide/cppguide.html#auto